e2e/tests/README.md

```diff
@@ -0,0 +1,40 @@
+e2e tests
+=========
+
+> tl; dr - `make localnet_up` and then `make test_e2e` 
+
```

Comment:
Consider using [1] to create a table of contents.

[1] https://marketplace.visualstudio.com/items?itemName=huntertran.auto-markdown-toc